Makefile

```diff
@@ -40,121 +40,88 @@ ifndef SR_XML
 	SR_XML=XE_SR_ERRORCODES.xml
 endif
 
-# OASIS_START
+build:
+	jbuilder build c/gen_c_binding.exe
+	jbuilder build csharp/gen_csharp_binding.exe
+	jbuilder build java/main.exe
+	jbuilder build powershell/gen_powershell_binding.exe
 
-SETUP = ocaml setup.ml
+release:
+	jbuilder build @install
 
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+install:
+	jbuilder install
```

Comment:
Does this do anything? It seems to me that this package has not jbuilder install targets